lcov.info

```diff
@@ -55,97 +73,110 @@ DA:0,4
 DA:21,1
 DA:26,1
 DA:35,0
-DA:43,0
-DA:51,1
-DA:53,1
-DA:122,0
-DA:124,0
-DA:133,1
-LF:10
+DA:50,1
+DA:52,1
+DA:121,0
+DA:123,0
+DA:132,1
+LF:9
 LH:6
-BRDA:7,1,0,4
-BRDA:7,1,1,4
-BRDA:7,2,0,4
-BRDA:7,2,1,0
-BRDA:7,3,0,1
-BRDA:7,3,1,0
-BRDA:7,4,0,0
-BRDA:7,4,1,1
-BRDA:27,5,0,0
-BRDA:27,5,1,4
-BRDA:27,6,0,4
-BRDA:27,6,1,4
-BRDA:29,7,0,0
-BRDA:29,7,1,1
-BRDA:31,8,0,0
-BRDA:31,8,1,1
-BRDA:31,9,0,0
-BRDA:31,9,1,1
-BRDA:31,10,0,1
-BRDA:31,10,1,0
-BRDA:31,10,2,0
-BRDA:33,11,0,0
-BRDA:33,11,1,1
-BRDA:33,12,0,1
-BRDA:33,12,1,0
-BRDA:33,13,0,1
-BRDA:33,13,1,1
-BRDA:33,14,0,1
-BRDA:33,14,1,0
-BRDA:33,15,0,0
-BRDA:33,15,1,1
-BRDA:253,16,0,0
-BRDA:253,16,1,0
+BRDA:8,1,0,4
+BRDA:8,1,1,4
+BRDA:8,2,0,4
+BRDA:8,2,1,0
+BRDA:8,3,0,1
+BRDA:8,3,1,0
+BRDA:8,4,0,0
+BRDA:8,4,1,1
+BRDA:28,5,0,0
+BRDA:28,5,1,4
+BRDA:28,6,0,4
+BRDA:28,6,1,4
+BRDA:30,7,0,0
+BRDA:30,7,1,1
+BRDA:32,8,0,0
+BRDA:32,8,1,1
+BRDA:32,9,0,0
+BRDA:32,9,1,1
+BRDA:32,10,0,1
+BRDA:32,10,1,0
+BRDA:32,10,2,0
+BRDA:34,11,0,0
+BRDA:34,11,1,1
+BRDA:34,12,0,1
+BRDA:34,12,1,0
+BRDA:34,13,0,1
+BRDA:34,13,1,1
+BRDA:34,14,0,1
+BRDA:34,14,1,0
+BRDA:34,15,0,0
+BRDA:34,15,1,1
+BRDA:251,16,0,0
+BRDA:251,16,1,0
 BRF:33
 BRH:31
 end_of_record
 TN:
 SF:/home/parikhv/WebstormProjects/eDocLite/src/reducers/todos.js
 FN:7,(anonymous_1)
-FN:9,_toConsumableArray
-FN:11,todo
-FN:32,todos
-FN:40,(anonymous_5)
-FNF:5
-FNH:5
-FNDA:1,(anonymous_1)
+FN:13,_toConsumableArray
+FN:18,todo
+FN:51,ADD_TODO
+FN:55,MOVE_TODO
+FN:56,(anonymous_6)
+FN:60,(anonymous_7)
+FN:64,findTodo
+FN:80,TOGGLE_TODO
+FN:81,(anonymous_10)
+FNF:10
+FNH:9
+FNDA:2,(anonymous_1)
 FNDA:3,_toConsumableArray
-FNDA:5,todo
-FNDA:5,todos
-FNDA:2,(anonymous_5)
-DA:0,5
-DA:1,1
-DA:2,5
-DA:4,3
-DA:10,2
-DA:11,1
-DA:14,1
-DA:19,0
-DA:23,1
-DA:24,5
-DA:26,3
-DA:31,1
-DA:35,1
-LF:13
-LH:12
+FNDA:7,todo
+FNDA:3,ADD_TODO
+FNDA:1,MOVE_TODO
+FNDA:2,(anonymous_6)
+FNDA:1,(anonymous_7)
+FNDA:0,findTodo
+FNDA:2,TOGGLE_TODO
+FNDA:4,(anonymous_10)
+DA:0,8
+DA:6,1
+DA:7,7
+DA:9,3
+DA:16,0
+DA:23,4
+DA:24,2
+DA:26,2
+DA:32,0
+DA:39,1
+DA:46,1
+DA:48,1
+DA:50,1
+DA:51,0
+DA:54,1
+DA:56,0
+DA:57,0
+DA:59,0
+DA:62,0
+DA:64,0
+DA:67,2
+LF:21
+LH:13
 BRDA:7,1,0,1
 BRDA:7,1,1,1
-BRDA:7,2,0,4
+BRDA:7,2,0,8
 BRDA:7,2,1,0
-BRDA:9,3,0,3
-BRDA:9,3,1,0
-BRDA:12,4,0,3
-BRDA:12,4,1,2
-BRDA:12,4,2,0
-BRDA:20,5,0,1
-BRDA:20,5,1,1
-BRDA:33,6,0,1
-BRDA:33,6,1,4
-BRDA:33,7,0,5
-BRDA:33,7,1,5
-BRDA:36,8,0,3
-BRDA:36,8,1,1
-BRDA:36,8,2,1
-BRF:18
-BRH:17
+BRDA:13,3,0,3
+BRDA:13,3,1,0
+BRDA:19,4,0,3
+BRDA:19,4,1,0
+BRDA:19,4,2,4
+BRDA:19,4,3,0
+BRDA:35,5,0,2
+BRDA:35,5,1,2
+BRDA:70,6,0,0
+BRDA:70,6,1,0
+BRF:14
+BRH:10
 end_of_record
```

Comment:
i did add lcov.info to .gitignore, but some how it didnt get sync. Will look into it and remove it from here.